tests/framework/client/client.go

```diff
@@ -0,0 +1,70 @@
+package client
+
+import (
+	"sync"
+
+	migrationsv1 "kubevirt.io/client-go/generated/kubevirt/clientset/versioned/typed/migrations/v1alpha1"
+
+	"kubevirt.io/kubevirt/tests/util"
+
+	corev1 "k8s.io/client-go/kubernetes/typed/core/v1"
+
+	"kubevirt.io/client-go/kubecli"
+)
+
+// TestClient
+//This client is a wrapped kubevirt client that exposes all client functionality
+//and can be used as a normal client. It provides a mechanism that track
+//non-namespaced resources created or updated during e2e tests and functions to rollback them at the end of the execution.
+
+type TestClient struct {
```

Comment:
nit: would change to:
```go
// TestClient is a wrapped kubevirt client that exposes all client functionality
// and can be used as a normal client. It provides a mechanism that track
// non-namespaced resources created or updated during e2e tests and functions to rollback them at the end of the execution.
type TestClient struct {
```

that is:
- "TestClient" is now part of the explanation
- added space between slashes and text
- removed empty line between doc and struct declarazation

Reply:
Great, thanks :+1: